cupy/linalg/product.py

```diff
@@ -7,6 +7,7 @@
 
 from cupy.linalg.solve import inv
 from cupy.util import collections_abc
+from cupy import util
```

Comment:
Please move the line one above, to keep the imports sorted alphabetically.

Reply:
@niboshi Fixed it